src/test/regress/expected/multi_sequence_default.out

```diff
@@ -981,8 +909,92 @@ SELECT nextval('seq_13');
+(0 rows)
+
+SELECT pg_identify_object_as_address(classid, objid, objsubid) from citus.pg_dist_object WHERE objid IN ('test_schema_for_sequence_default_propagation'::regnamespace);
+ pg_identify_object_as_address
+---------------------------------------------------------------------
+(0 rows)
+
+-- Create distributed table with default column to propagate dependencies
+CREATE TABLE test_seq_dist(a int, x BIGINT DEFAULT nextval('test_schema_for_sequence_default_propagation.seq_10'));
+SELECT create_distributed_table('test_seq_dist', 'a');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+-- Both sequence and dependency schema should be distributed
+SELECT pg_identify_object_as_address(classid, objid, objsubid) from citus.pg_dist_object WHERE objid IN ('test_schema_for_sequence_default_propagation.seq_10'::regclass);
+                     pg_identify_object_as_address
+---------------------------------------------------------------------
+ (sequence,"{test_schema_for_sequence_default_propagation,seq_10}",{})
+(1 row)
+
+SELECT pg_identify_object_as_address(classid, objid, objsubid) from citus.pg_dist_object WHERE objid IN ('test_schema_for_sequence_default_propagation'::regnamespace);
+               pg_identify_object_as_address
+---------------------------------------------------------------------
+ (schema,{test_schema_for_sequence_default_propagation},{})
+(1 row)
+
+-- Show that sequence can stay on the worker node if the transaction is
+-- rollbacked after distributing the table
+BEGIN;
+CREATE SEQUENCE sequence_rollback;
+CREATE TABLE sequence_rollback_table(id int, val_1 int default nextval('sequence_rollback'));
+SELECT create_distributed_table('sequence_rollback_table', 'id');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+ROLLBACK;
+-- Show that there is a sequence on the worker with the sequence type int
+\c - - - :worker_1_port
+SELECT seqtypid::regtype, seqmax, seqmin FROM pg_sequence WHERE seqrelid::regclass::text = 'sequence_rollback';
+ seqtypid |   seqmax   | seqmin
+---------------------------------------------------------------------
+ integer  | 2147483647 |      1
+(1 row)
+
+\c - - - :master_port
+-- Show that we can create a sequence with the same name and different data type
+BEGIN;
+CREATE SEQUENCE sequence_rollback;
+CREATE TABLE sequence_rollback_table(id int, val_1 bigint default nextval('sequence_rollback'));
+SELECT create_distributed_table('sequence_rollback_table', 'id');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+ROLLBACK;
+-- Show that existing sequence has been renamed and a new sequence with the same name
+-- created for another type
+\c - - - :worker_1_port
+SELECT seqrelid::regclass, seqtypid::regtype, seqmax, seqmin FROM pg_sequence WHERE seqrelid::regclass::text like '%sequence_rollback%';
```

Comment:
very minor: can you add ORDER BY? We have seen lots of flaky tests even for such things :/